src/util/log.cc

```diff
@@ -269,11 +278,11 @@ logger::do_log(log_level level, log_writer& writer) {
       return;
     }
     static array_map<sstring, 20> level_map = {
-            { int(log_level::debug), "DEBUG" },
-            { int(log_level::info),  "INFO "  },
-            { int(log_level::trace), "TRACE" },
-            { int(log_level::warn),  "WARN "  },
-            { int(log_level::error), "ERROR" },
+            { int(log_level::debug), COLORED(fmt::color::white,  "DEBUG") },
```

Comment:
does it work if terminal has white background?

Reply:
no. i changed to use the "terminal_color" . see https://github.com/fmtlib/fmt/blob/69a20db081e3b4434e2b8ef7e8a5a37a0647b391/include/fmt/color.h#L160 

so only the ANSI colors are used, and if user are using a color theme which maps ANSI colors to another color in the terminal, the corresponding color is used instead of hardcoded RGB color.